SPECS/rust/rust.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Rust Programming Language
 Name:           rust
-Version:        1.50.0
+Version:        1.47.0
```

Comment:
I think we should support both versions, rather than downgrade

Reply:
Even better if we can patch whatever the problem is with 1.50 and IoT-edge

Reply:
It's not really a downgrade, since we have version 1.39.0 checked in. After discussing with iotedge team - rust 1.47.0 is confirmed to work properly with azure-iotedge